Resources/Prototypes/_FarHorizons/Entities/Objects/Devices/pda.yml

```
@@ -115,3 +115,130 @@
       whitelist:
         tags:
         - Write
+
+#SYNDI OPERATIVE PDA - needed for nukie pdas so they dont have sensors that show crew where they are
```

Since we will be having NT Nuke Ops in the future it would be good to have the name on this neutral.

```suggestion
#Nuclear Operatives PDA - Needed for nukie pdas so they dont have sensors that show crew where they are
```

---

funnily enough i actually named it Nukeops pda first but felt that Syndie operative made more sense but ye sure